tests/tensorflow/test_models_zoo_tf.py

```diff
@@ -91,6 +92,9 @@ def test_trained_ocr_predictor(mock_tilted_payslip):
                                  [0.51385817, 0.21002172]])
     assert np.allclose(np.array(out.pages[0].blocks[1].lines[0].words[-1].geometry), geometry_revised)
 
+    orientation = 30.415687561035156
```

Comment:
@frgfm @charlesmindee

Reply:
this value seems to be wrong

Reply:
a holy now i got it :sweat_smile: @aminemindee please move this test inside `test_ocrpredictor` :+1:

Reply:
Done and pushed. Yeah it's better i used there in pytorch but not on tf haha my bad